eslint.config.mjs

```diff
@@ -45,7 +45,6 @@ export default tseslint.config([
 
             "react/display-name": 0,
             "react/no-children-prop": 0,
-            "react/no-unknown-property": 0,
```

Comment:
did you mean to still remove this? Looks like you fixed the issues it related to so maybe then leave it in place?

Reply:
Yes, my intent was to remove it - since it currently turns the rule off. It is turned on by the recommended react eslint rules.